lib/rest2.js

```diff
@@ -381,6 +382,16 @@ class RESTv2 {
     return this._makePublicRequest(`/book/${symbol}/${prec}`, cb)
   }
 
+  /**
+   * @param {string} nickname - i.e. Bitfinex
+   * @param {Function} [cb] - callback
+   * @returns {Promise} p
+   * @see https://docs.bitfinex.com/reference#rest-public-pulse-profile
+   */
+  publicPulseProfile (nickname = 'Bitfinex', cb) {
```

Comment:
Lets leave out the default parameter here. Imo its best that the function/request fails here rather than an unexpected result being returned